src/transport/GasTransport.cpp

```diff
@@ -357,6 +357,16 @@ void GasTransport::setupCollisionParameters()
     }
 }
 
+void GasTransport::invalidateCache()
+{
+    Transport::invalidateCache();
+    m_temp += 0.1234;
```

Comment:
I know we have values like this elsewhere, but could we use something more obvious regarding exception handling? Absent better options, I prefer the '-1' value that was used before.